ext/standard/tests/general_functions/proc_open_multiplex.phpt

```diff
@@ -0,0 +1,51 @@
+--TEST--
+Multiplexing of child output
+--FILE--
+<?php
+$php = getenv("TEST_PHP_EXECUTABLE");
+$desc = [
+    ["null"],
+    ["pipe", "w"],
+    ["null"]
+];
+$read_pipes = [];
+for ($i = 0; $i < 10; $i++) {
+    $procs[] = proc_open([$php, "-r", "usleep(100000 * (10 - $i)); echo 'hello$i';"], $desc, $pipes);
```

Comment:
Does it really need to take the whole second (10 x 100ms) to test this? I think it should work for shorter interval to test what is tested here.

Reply:
I've reduced to 10 * 10ms, and CI is still green.